articles/building-documentation-websites-with-docusaurus-v2/index.md

```diff
@@ -0,0 +1,201 @@
+## Building  documentation websites with Docusaurus version 2
+
+### Introduction
+
+Have you always wanted to start your blog? Are you working on a project and not sure how to go about the documentation? Well, Docusaurus is an open-source project for building, deploying, and maintaining documentation websites.
+
+Docusaurus is maintained by the Facebook Open Source community and there is a lot of support around it, so you don't have to worry if something goes wrong during development. For this tutorial, we are going to use Docusaurus version 2.
+
+### What are the benefits of Docusaurus?
+
+There are a lot of static site-generating projects available today, but what makes Docusaurus version 2 unique?
+
+**Highly customizable** - Docusaurus is very customizable, it can be used for pretty much anything to do with static websites including landing pages since it is built with React.js. So you don't have to worry if your page is going to be unique enough.
+
+**SEO friendly** - Page-specific SEO to help users land on the official documentation site directly relating their problems at hand.
+
+**Easy to setup** - Docusaurus does not require you to be a seasoned programmer for you to set it up. As long as you can follow instructions and curious enough to try, then you are set to go.
+
+### Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/) version 12.13.0 or above installed
+
+- [Yarn](https://classic.yarnpkg.com/en/docs/install/#debian-stable) version 1.5 installed
+
+### Installation
+
+Now that you are ready to build your site let us start with the installation. Docusaurus can be installed as an npm package on the command line interface.
+
+```bash
+npx @docusaurus/init@latest init [name] [template]
+```
+
+For the ***name*** part, you are going to use your preferred name for this project.
+
+For the ***template*** enter the name of the template you are going to use. Docusaurus comes with three templates Classic, Facebook, and Bootstrap. The Facebook template is used for Facebook open-source projects and comes with Facebook-specific defaults, bootstrap template is used when you want the bootstrap framework website. For this tutorial, we are going to use the classic template.
+
+Run the following command.
+
+```bash
+npx @docusaurus/init@latest init my-blog classic
+```
+
+### File structure
+
+The command generates a directory known as my-blog. Navigate into the directory to see the generated files. Some of the files and folders that you should pay attention to are `docusaurus.config.js`, the `src` folder, and the `blog` folder. Let us run and build the project and see how it looks like without any customization.
+
+```bash
+npm run start
+```
+
```

Comment:
Please explain the difference between `npm run start` and `npm run build` in this context.